virttest/qemu_vm.py

```diff
@@ -3839,18 +3839,22 @@ def create(
                 self.destroy()
                 raise virt_vm.VMStartError(self.name, err)
 
+            def _check_vm_alive():
+                if not self.process.is_alive():
```

Comment:
Looks we just need to put the condition check into the `wait_for` instead of the entire if clause.

```python
if not wait_for(self.process.is_alive, 5):
    ...
```

Reply:
If it just checks one time, it is ok. but the same checking needs to be done when monitoring access timeout.

Reply:
If we want to check at one place, it may consider to add check time, it could be like this :

if utils_misc.wait_for(lambda:not self.process.is_alive(), 8, 3):

What do you think ?